Sources/FirestoreManager/FirestoreCRUD/Update.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FirestoreManager+Update.swift
+//  
+//
+//  Created by Anna Münster on 26.08.21.
+//
+
+import Foundation
+import FirebaseFirestore
+
+extension FirestoreManager {
+    /**
+     Update a document in Firebase.
+     
+     - Parameter object: Codable object to be uploaded
+     - Parameter reference: The collection name
+     - Parameter id: ID to be set for the document
+     */
+    public static func updateDocument<T>(_ object: T,
+                                         reference: ReferenceProtocol,
+                                         with id: String,
+                                         completion: @escaping(Result<String, FirestoreError>) -> Void) where T: Encodable {
+        
+        do {
+            try reference.reference().document(id).setData(from: object, merge: false)
+            completion(.success(id))
+        } catch {
+            completion(.failure(.update(error: error)))
+        }
+    }
+    
+    public static func batchWrite<T>(_ data: [T], reference: ReferenceProtocol, parentId: String? = nil, completion: @escaping(FirestoreError?) -> Void) where T: Encodable {
```

Comment:
Does this batch create or batch update? I think documentation for the function would be good, also to state what the parentId is, as this parameter isn't used somewhere else and might be confusing.